android/build.gradle

```diff
@@ -54,7 +54,7 @@ android {
      * The obvious way to fix this would be allow wildcards (ie, "22.+") but
      * Android have explicitly said that they won't do this.
      */
-    compileSdkVersion 22
-    buildToolsVersion '21.1.2'
+    compileSdkVersion 25
```

Comment:
The commit message does not mention anything about changing the SDK version. Is this change intentional? If it is, it should be explained the the commit message why it's being done.